frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -210,116 +213,123 @@ export const FieldRowContainer = ({
           {...provided.draggableProps}
           ref={provided.innerRef}
         >
-          <Flex
-            // Offset for focus boxShadow
-            my="2px"
-            // Focusable
-            tabIndex={0}
-            role="button"
-            cursor={isActive ? 'initial' : 'pointer'}
-            bg="white"
-            transition="background 0.2s ease"
-            _hover={{ bg: isDraggingOver ? 'white' : 'secondary.100' }}
-            borderRadius="4px"
-            outline="none"
-            {...(isActive ? { 'data-active': true } : {})}
-            _focusWithin={{
-              boxShadow: snapshot.isDragging
-                ? 'md'
-                : '0 0 0 2px var(--chakra-colors-primary-500) !important',
-            }}
-            _active={{
-              bg: 'secondary.100',
-              boxShadow: snapshot.isDragging
-                ? 'md'
-                : '0 0 0 2px var(--chakra-colors-primary-500)',
-            }}
-            flexDir="column"
-            align="center"
-            onClick={handleFieldClick}
-            onKeyDown={handleKeydown}
-            ref={ref}
+          <Tooltip
+            hidden={!isHiddenByLogic}
+            placement="top"
+            label="This field may be hidden by your form logic"
```

Comment:
`may be` or `is`? Asking because the Tooltip's only control on its hidden state is `isHiddenByLogic` 🤔

Reply:
hahah. this was the copy that was agreed upon. In actual fact, the field is hidden by logic by default (i.e. on first load of the form). Could change the variable name to `isDefaultHiddenByLogic` if that feels better?

Reply:
Nah, let's keep it. We can revisit later if needed.

Reply:
may is correct, since a logic match will show the field yea?